code/game/gamemodes/clock_cult/clock_scriptures/scripture_revenant.dm

```diff
@@ -136,12 +136,14 @@
 	nezbere_invoked++
 	for(var/obj/O in all_clockwork_objects)
 		O.ratvar_act()
-	spawn(600)
-		nezbere_invoked--
-		for(var/obj/O in all_clockwork_objects)
-			O.ratvar_act()
+	addtimer(CALLBACK(src, .proc/reset_invocation), 600)
```

Comment:
This won't work, smart guy. The scripture doesn't EXIST after a minute, you're gonna need to make it a global proc.